tox.ini

```diff
@@ -1,14 +1,21 @@
 [tox]
 envlist =
   py27
-  py34
+  py35
```

Comment:
Should we either add py34 here or remove it from the Dockerfile?

Reply:
Digital Marketplace (one of the few other users of the Python client) are running 3.6 in production. 

Let’s deal with this as a separate PR if we want to test against multiple versions. 3.5 is the best version to test against if we have to pick only one.